src/scales/scale.category.js

```diff
@@ -53,10 +53,15 @@ module.exports = function() {
 
 		getLabelForIndex: function(index, datasetIndex) {
 			var me = this;
-			var data = me.chart.data;
+			var chart = me.chart;
+			var data = chart.data;
 			var isHorizontal = me.isHorizontal();
+			var ds = typeof chart.getDatasetMeta === 'function' && chart.getDatasetMeta(datasetIndex).controller;
```

Comment:
Why do we need `typeof chart.getDatasetMeta === 'function'`?  Is `getDatasetMeta` not always present? When would that be?

Reply:
[The way](https://github.com/chartjs/Chart.js/blob/c8ce3d2018244657f18f7bae2fd2fcf1fe825e8f/test/specs/scale.category.tests.js#L94) tests are written without actual chart.